docs/server-configurations.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Server Configurations
+category: 62cc6ce22b8b6601da6cb12d
+---
+
+If the API you're using offers alternate server URLs and server variables in its [`servers`](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#serverObject) definition you can supply this to the SDK with `.server()`:
+
+```js
+sdk.server('https://{region}.api.example.com/{basePath}', {
+  name: 'eu',
+  basePath: 'v14',
+});
```

Comment:
Product feedback—it'd be nice if you could just supply a regular ol' string here and it would work (provided it adhered to the server variable constraints):
```js
sdk.server('https://eu.api.example.com/v14');
```